deploy/release.groovy

```diff
@@ -66,9 +66,9 @@ def cd (b){
             container('ui'){
                 sh '''
         npm cache clean --force
+        npm cache verify
         npm install
-        cd src/tests/functionalTests
-        DEBUG=true HEADLESS_MODE=true ./run_ts_functional_tests.sh smokeTest
+        DEBUG=true HEADLESS_MODE=true ./scripts/run-functests.sh smokeTest
```

Comment:
The `smokeTest` argument won't make any difference. The `run-functests.sh` script doesn't pass the argument to `run.sh` script

Reply:
Fixed. See below.